source/leuven/wice_advanced_guide.rst

```diff
@@ -0,0 +1,121 @@
+.. _wice_t2_leuven_advanced:
+
+===================
+wICE advanced guide
+===================
+
+.. _wice_compilation:
+
+Compiling software
+------------------
+
+The wICE compute nodes feature Intel processors nicknamed IceLake, which are
+the successors of the SkyLake and CascadeLake architectures which you can find
+on Genius. Although architectural differences are rather small, it is highly
+recommended to compile a version of your code specifically for wICE. Since the
+operating system (Rocky Linux 8.6) differs significantly from Genius, a
```

Comment:
hard-coded version(s) need continuous updating of docs per each version change.
One safe approach is to say, e.g. `8.6+` or `>8.6` or any other format you wish.